### PR TITLE
MSGRPC module minor fixes

### PR DESCRIPTION
**Pull request by TheNaterz**
This PR takes care of two issues that are currently present with the MSGRPC module when loading, unloading, then reloading:

1. Current behavior produces two warnings upon reloading about the already initialized constants DefaultHost and DefaultPort from the first load. This has been corrected using OR Equal.
2. Current behavior does not start the HTTP server upon reloading. The HTTP server is never dereferenced from the ServiceManager, so it thinks that the service still exists and doesn't run the HTTP server instance. This has been corrected by explicitly calling service.deref.

In short, go ahead and load/unload msgrpc as many times as you want now :)

### CODE REVIEW

**Comment by bcoles** [comment]
I've verified this fixes #8561 

```
msf > load msgrpc ServerHost=127.0.0.1 Pass=abc123 SSL=y
[*] MSGRPC Service:  127.0.0.1:55552  (SSL)
[*] MSGRPC Username: msf
[*] MSGRPC Password: abc123
[*] Successfully loaded plugin: msgrpc
msf > netstat -an | grep 55552
[*] exec: netstat -an | grep 55552

tcp        0      0 127.0.0.1:55552         0.0.0.0:*               LISTEN     
msf > unload msgrpc
Unloading plugin msgrpc...unloaded.
msf > netstat -an | grep 55552
[*] exec: netstat -an | grep 55552

msf > load msgrpc ServerHost=127.0.0.1 Pass=abc123 SSL=y
[*] MSGRPC Service:  127.0.0.1:55552  (SSL)
[*] MSGRPC Username: msf
[*] MSGRPC Password: abc123
[*] Successfully loaded plugin: msgrpc
msf > netstat -an | grep 55552
[*] exec: netstat -an | grep 55552

tcp        0      0 127.0.0.1:55552         0.0.0.0:*               LISTEN     
```

**Comment by busterb** [comment]
Thanks guys, you are great!

**Comment by busterb** [comment]
## Release Notes

This fixes a couple of issues with the MSGRPC plugin, allowing it reload without error.
